### PR TITLE
[RND-578] - Integrates Elastic Search tests into CI process.

### PR DESCRIPTION
**Pull request by DavidJGapCR**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have signed all of the commits on this PR.
- [ ] I have agreed to the Ed-Fi Individual Contributors' License
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Meadowlark-js/tests/config/global-setup.js [hunk 1]
```diff
@@ -1 +1 @@
-jest.setTimeout(40000)
+jest.setTimeout(100000)
```
Comment (andonyns) [inline Meadowlark-js/tests/config/global-setup.js:1]: Why was it necessary to increase the default timeout?

File: Meadowlark-js/tests/e2e/setup/EnvironmentConfig.ts [hunk 2]
```diff
@@ -7,17 +7,27 @@ import { Network, StartedNetwork } from 'testcontainers';
...
-  await Promise.all([MongoContainer.stop(), ApiContainer.stop(), OpenSearchContainer.stop()]);
+  await Promise.all([MongoContainer.stop(), ApiContainer.stop()]);
+
   if (process.env.DOCUMENT_STORE_PLUGIN === '@edfi/meadowlark-postgresql-backend') {
     console.info('-- Tearing down postgres --');
     await PostgreSqlContainer.stop();
   }
+
+  if (process.env.QUERY_HANDLER_PLUGIN === '@edfi/meadowlark-elasticsearch-backend') {
+    console.info('-- Setting up elasticsearch --');
```
Comment (andonyns) [inline Meadowlark-js/tests/e2e/setup/EnvironmentConfig.ts:25]: This message should be **Tearing down**

File: Meadowlark-js/tests/e2e/setup/EnvironmentConfig.ts [hunk 3]
```diff
@@ -7,17 +7,27 @@ import { Network, StartedNetwork } from 'testcontainers';
...
+    await ElasticSearchContainer.stop();
+  } else {
+    console.info('-- Setting up opensearch --');
```
Comment (andonyns) [inline Meadowlark-js/tests/e2e/setup/EnvironmentConfig.ts:28]: This message should be **Tearing down**

File: Meadowlark-js/tests/config/global-setup.js (hunk 1 above)
Comment (DavidJGapCR) [reply Meadowlark-js/tests/config/global-setup.js]: Undoing this change.

**Review by andonyns — APPROVED** [review]

**Review by bradbanister — APPROVED** [review]

**Pull request merged**